ovos_bus_client/scripts.py

```diff
@@ -22,6 +23,7 @@ def ovos_speak():
     if not client.connected_event.is_set():
         client.connected_event.wait()
     client.emit(Message("speak", {"utterance": utt, "lang": lang}))
+    time.sleep(0.5)  # avoids crash in c++ bus server
```

Comment:
Don't love this hack, don't have a better suggestion

Reply:
One way to find out!

Reply:
since these are just cli scripts, i dont really care if they take .5 seconds longer to exit... if it makes things more stable leave them in

Reply:
It's a slightly annoying DX, nothing worth blocking a release for. I was wondering why those commands always lagged